src/Aspire.ProjectTemplates/templates/aspire-starter/AspireStarterApplication.1.ApiService/AspireStarterApplication.1.ApiService.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
```

Comment:
Is this right? Same question for the Service Defaults change removing Library (which that one I do believe the default is Library so probably fine.)

Reply:
`Microsoft.NET.Sdk.Web` defaults OutputType to Exe. No ASP.NET core templates explicitly set OutputType.

https://github.com/dotnet/sdk/blob/380a3c12642d885f540f0e88bcd611dae887a97e/src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.props#L16

https://github.com/dotnet/aspnetcore/blob/2dc2ab32eddb6ea697824d89fa6c854bfe13463f/src/ProjectTemplates/Web.ProjectTemplates/WebApi-CSharp.csproj.in#L1-L10